data/en/parameterexists.json

```diff
@@ -4,12 +4,12 @@
 	"syntax":"parameterExists(name)",
 	"returns":"boolean",
 	"related":[],
-	"description":"This function is deprecated. Use the IsDefined or StructKeyExists functions. Determines whether a parameter exists. ColdFusion does not evaluate the argument.",
```

Comment:
Since this is already in engine notes, you can remove it from the function's description.